src/monitors/extractLineMonitor.cpp

```diff
@@ -33,64 +33,64 @@ void ablate::monitors::ExtractLineMonitor::Register(std::shared_ptr<solver::Solv
     // get the min cell size
     PetscReal minCellRadius;
     DMPlexGetGeometryFVM(flow->GetSubDomain().GetDM(), NULL, &cellGeomVec, &minCellRadius) >> checkError;
-    VecGetDM(cellGeomVec, &dmCell) >> checkError;
+    VecGetDM(cellGeomVec, &dmCell) >> checkError; //Gets the DM defining the data layout of the vector
     VecGetArrayRead(cellGeomVec, &cellGeomArray) >> checkError;
 
     PetscMPIInt rank;
     MPI_Comm_rank(flow->GetSubDomain().GetComm(), &rank) >> checkMpiError;
 
     PetscInt dim;
-    DMGetDimension(flow->GetSubDomain().GetDM(), &dim) >> checkError;
-
-    // Now march over each sub segment int he line
-    double ds = minCellRadius / 10.0;
-    double s = 0.0;
-    double L = 0.0;
-    std::vector<double> lineVec;
-    for (std::size_t d = 0; d < start.size(); d++) {
-        L += PetscSqr(end[d] - start[d]);
-        lineVec.push_back(end[d] - start[d]);
+    DMGetDimension(flow->GetSubDomain().GetDM(), &dim) >> checkError; //Gets the number of dimensions which the domain exists in?
+
+    // Now march over each subsegment in the line
+    double ds = minCellRadius / 10.0; //Sets the minimum step size which the line monitor is allowed to take
+    double s = 0.0; //s represents how far the line has been marched through?
+    double L = 0.0; //L represents the length of the line?
+    std::vector<double> lineVec; //Vector in the direction of the line
+    for (std::size_t d = 0; d < start.size(); d++) { //
+        L += PetscSqr(end[d] - start[d]); //PetscSqr: squares a number?
+        lineVec.push_back(end[d] - start[d]); //What is push_back? Adds a new element to the vector
     }
-    L = PetscSqrtReal(L);
+    L = PetscSqrtReal(L); //Takes the square root of the sum of differences?
     for (auto& c : lineVec) {
-        c /= L;
+        c /= L; //TODO: What does c represent?
```

Comment:
TODO: has a meaning in most ides.  Would you mind updating the comments so that we can remove the todo's?